docs/content/docs/dev/table/sqlClient.md

```diff
@@ -601,6 +601,24 @@ This configuration:
 
 <span class="label label-danger">Attention</span> Compared to the interactive mode, SQL Client will stop execution and exits when there are errors.
 
+### Deploy SQL Files in an Application Cluster
```

Comment:
Deploy SQL Files to an Application Cluster  Or Deploy SQL Files in Application Mode  ?